guides/v2.3/contributor-guide/backward-compatible-development/index.md

```diff
@@ -443,6 +444,12 @@ For example, when 2.2 is released, a new `backward-incompatible-changes.md` for
 
 In order to update the page, create a PR to the DevDocs repository with your changes.
 
+## GraphQl backward compatibility policy
+
+* In GraphQL, the schema is the only code that can be counted as a contract.
+* Only the schema should be taken into account for backward compatibility.
+* The GraphQL codebase should NOT follow the restrictions described in this topic. For example, skip all the `backward compatibility` restrictions for the GraphQL PHP codebase.
```

Comment:
The only exception we make for GraphQL is we allow MINOR code changes in PATCH Magento releases. Any MAJOR changes are not permitted in PATCH releases. This includes MAJOR changes in schema as well as in the codebase. In the codebase we do not have GraphQL-related classes marked with `@api` intentionally because the project is still in the development phase and we do not want limit refactoring capabilities.

GraphQL schema is considered `@api`.